src/actions/index.js

```diff
@@ -52,4 +52,16 @@ export const addProject = project => ({
 export const removeProject = id => ({
   type: 'REMOVE_PROJECT',
   id
-})
\ No newline at end of file
+})
+
+export const resetSelectedPalette = () => ({
+  type: 'RESET'
```

Comment:
can you have actions without payloads?

Reply:
Good call. I could just return my desired object in the action